lib/dialogs/buy/bloc/BuyDialogBloc.dart

```diff
@@ -22,6 +22,7 @@ class BuyDialogBloc extends Bloc<BuyDialogEvent, BuyDialogState> {
     on<OnMaxBuyTap>(_mapMaxBuyTapEventToState);
     on<OnConfirmBuy>(_mapConfirmBuyEventToState);
     on<OnNewAxInput>(_mapNewAxInputEventToState);
+    on<OnErrorEvent>(_errorEventState);
```

Comment:
If the event arises from an error coming from a dependency of this class that should be propagated down to the bloc then the bloc will update the state. 

As these events are meant to communicate to the bloc from the widget this is not the correct way to do this fix.